airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -33,31 +34,49 @@ class ShopifyBulkManager:
     session: requests.Session
     base_url: str
 
-    # 5Mb chunk size to save the file
-    retrieve_chunk_size: int = 1024 * 1024 * 5
+    # default logger
+    logger: ClassVar[AirbyteLogger] = logging.getLogger("airbyte")
```

Comment:
I like the idea of annotating those as `ClassVar` as it clearly shows that those are the same for each instance. Should we go a bit further and set them as [Final](https://docs.python.org/3/library/typing.html#typing.Final) as they are read only?

Reply:
Done